packages/terra-form-input/src/FormInput.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
```

Comment:
Nit: After reading the read me I would have expected this file to be named `Input.jsx`
```
import Input from 'terra-form-input';

<Input
  name="description"
  value="I'm a software engineer working at Cerner"
  required
/>
``` 
With the ability to alias it may not make too much of a difference, do you have a preference either way?

Reply:
Let's rename this file to `Input.jsx` With all of the other new terra-form-component-name packages, the JSX file is named without the `Form` prefix. e.g. Radio.jsx, Select.jsx, Field.jsx

Reply:
I left these as they're what the [generator-terra-module](https://github.com/cerner/generator-terra-module) output automatically. Updated them in 0fa82cf1

Reply:
Renamed 0fa82cf1ef4966c97404188b2b229c8f8385fb2b